client/client.go

```diff
@@ -140,6 +140,11 @@ func (c *Client) On(eventType EventType, h EventHandler) {
 	c.handlers[eventType] = h
 }
 
+// Send sends a websocket event to the server
+func (c *Client) Send(ev string, msg any, binary bool) error {
+	return c.wsSend(ev, msg, binary)
+}
```

Comment:
I'd probably make it explicit this is a websocket send because this client wraps a bunch of stuff including RTC and potentially a data channel. A simple suffix should work.

Reply:
Yep, makes sense, thanks.